### DISCUSSION TITLE
Can image be an array of ImageObjects?

### DISCUSSION THREAD

**Discussion by do-kevin** [schemaorg/schemaorg#3521] [Q&A]
I'm receiving an error message from Google Search saying my image has an incorrect value. Here's a code snippet of what I had:

```
 {  "@context": "https://schema.org",
  "@type": "BlogPosting",
"mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "my_website_url"
  },
  "headline": "Hello World",
"image": [
    {
      "@type": "ImageObject",
      "contentUrl": "my_image_url1"
    },
    {
      "@type": "ImageObject",
      "contentUrl": "my_image_url2"
    },
    {
      "@type": "ImageObject",
      "contentUrl": "my_image_url3"
    },
    {
      "@type": "ImageObject",
      "contentUrl": "my_image_url4"
    },
    {
      "@type": "ImageObject",
      "contentUrl": "my_image_url5"
    }
  ]
}
```

```
 {  "@context": "https://schema.org",
  "@type": "BlogPosting",
"mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "my_website_url"
  },
  "headline": "Hello World",
"image": [
 "my_image_url", "my_image_url2", "my_image_url3"]
}
```

I've ran my json snippet inside Schema.org and Google's Rich Results Test, and they came up fine. But when Google Search tests my URL, it says image has an incorrect value.

I also see an example of an image array in Google Search Central: https://developers.google.com/search/docs/appearance/structured-data/article

So is `"image": []` valid or not?

**Comment by jvandriel** [comment]
Hi @do-kevin,

Providing an array of `ImageObject` nodes is the way it's supposed to be done. The error message you describe implies you have either used a wrong property label (e.g. `my_image_url`) or a false value (e.g. `my_image_url`).

**Reply by do-kevin** [reply]
Thank you,

I'm currently investigating, but it seems like escape characters are somehow added into the image array. They could be my culprits.
